analyzer/codechecker_analyzer/analysis_manager.py

```diff
@@ -627,6 +648,9 @@ def __create_timeout(analyzer_process):
                     handle_failure(source_analyzer, rh, zip_file,
                                    result_base, actions_map)
 
+        collect_ctu_involved_files(rh, source_analyzer,
```

Comment:
Do we want this collection even if the analysis was successful?

Reply:
Yes, we do. These files have to be used for replicating false-positive reports.

Reply:
Ok, please add a comment here why the collection is done even if the analysis was successful.